Add .NET hebe implementation

MRmlik12 opened the pull request.

mklkj commented on hebe-dotnet/UonetRequestSignerHebe/Signer.cs:
```diff
@@ -0,0 +1,67 @@
+{
+    public class Signer
+    {
+        public SignerValues GetSignatureValues(string fingerprint, string privKey, string body, DateTime date, string path)
```
Please use the same args order as in other implementations (swap last two): https://github.com/wulkanowy/uonet-request-signer/blob/master/hebe-jvm/src/main/kotlin/signer.kt#L55-L60

mklkj commented on hebe-dotnet/UonetRequestSignerHebe/Signer.cs:
```diff
@@ -0,0 +1,67 @@
+using System.Security.Cryptography;
+using System.Collections.Generic;
+
+namespace UonetRequestSignerHebe
```
Change namespace to `Wulkanowy.UonetRequestSigner.Hebe

mklkj reviewed (changes requested):
Please also rebase your branch

MRmlik12 commented:
Ok, I renamed solution to `Wulkanowy.UonetRequestSigner.Hebe` and updated `Signer.cs` args order. I deleted `SignerValues.cs` file because I used c# 7 feature it's called "value tuples".

MRmlik12 commented:
I modified Signer.cs with GetEncodedPath method to encode path properly

mklkj commented on hebe-dotnet/README.md:
````diff
@@ -0,0 +1,16 @@
+```cs
+using Wulkanowy.UonetRequestSigner.Hebe;
+
+var signed = new Signer().GetSignatureValues("fingerprint", "private_key", "body", "url", DateTime.Now);
````
One last thing - maybe make the `GetSignatureValues` method `static`? Similar to `dotnet` impl

MRmlik12 replied:
I'll try make a static class If you want
